_posts/fr/2020-10-16-ALTF4-9.md

```diff
@@ -68,9 +68,7 @@ Il est fascinant de constater à quel point ce mot particulier s'est naturelleme
 
 De vastes étendues d'herbe, de dangereux déserts rouges et des lacs infiniment profonds. Une terre riche en ressources mais habitée par une menace locale de déchiqueteurs. C'est l'endroit que nous avons appris à connaître et à aimer après nous être brutalement écrasés sur elle. C'est notre maison, et nous l'appelons Nauvis.
 
-![Image du mod Nauvis Day](https://media.alt-f4.blog/ALTF4/9/nauvis_day.jpg)
-
-<image-caption>Illustration du mod <a href="https://mods.factorio.com/mod/NauvisDay">Nauvis Day</a>, avec quelques retouches par l'auteur.</image-caption>
+{% include figure.html src='https://media.alt-f4.blog/ALTF4/9/nauvis_day.jpg' alt='Image du mod Nauvis Day' caption='Illustration du mod <a href="https://mods.factorio.com/mod/NauvisDay">Nauvis Day</a>, avec quelques retouches par l&#39;auteur.' %}
```

Comment:
this is annoying that some language have to escape the ' character with &#39 ... no other possibilities on the technical side?

Reply:
check #243 with the Italian having 3 escape characters

Reply:
Issue is that we have html link inside this string. and as far as I'm aware this require quotes `"`.
because of it we need to use apostrophes `'` as outer string identifier (not the best word to describe it, but i couldn't come up with better alternative). And because of it we need to escape internal apostrophes `'`

in regular cases (no link) caption would look like this 
```
... caption="Guntet O'Dim"
```
with quotes outside and no need to escape apostrophes

Reply:
ok, will try to remember :)